### ISSUE TITLE
Fallbacks don't appear to be working correctly.

### ISSUE THREAD

**Issue by Senjai** [fnando/i18n-js#297]
```yaml
translations:
- file: "public/javascripts/translations.js"
  only: "*.js"
```

Is my i18n-js file. In my en.yml, I have js.test defined, in my it.yml, I dont have js defined at all. In the developer console:
```js
I18n.t("js.test")
> "This is a test"
I18n.locale = "it"
> "it"
I18n.t("js.test")
> "[missing "it.js.test" translation]"
```

I would expect the last method call to yield the same as the first. Could anyone shed any light on this?

I have tried setting fallbacks to a locale manually in the configuration file to no avail as well.

**Comment by PikachuEXE** [comment]
Thanks for reporting.
Could you create a repo with minimum code to reproduce the option so we can start debugging?
It's much faster than asking for the environment like other gem versions, config, etc.
Thanks :)

**Comment by Senjai** [comment]
I can attempt to do this. I mostly wanted to drop the issue by in case I was expecting something I shouldn't be. I'm fairly busy so I cannot promise I can fulfill your request. I will attempt to find the time on the weekend but I cannot guarantee it.

I understand that its a lot more helpful for reporters to provide this kind of information, but even if I cant provide a repo, I still think its best to open an issue as you find them either way.

Cheers

**Comment by PikachuEXE** [comment]
I am a busy man too (until mid-March)
So I might be able to handle this issue (or most issues) during this period.
Normally I will just create that minimum code myself

**Comment by Senjai** [comment]
It's a long weekend this weekend, so I should be able to get something up.

**Comment by johnnyshields** [comment]
I've fixed this in PR #320. Please try it and give thumbs up if it works.

**Comment by johnnyshields** [comment]
I'm closing this issue as I've fixed it and the requester hasn't replied in 30 days.

**Comment by johnnyshields** [comment]
oops don't have access to close. @PikachuEXE can you please close this?

**Comment by PikachuEXE** [comment]
Closing since fixed in #320, reopen if issue still exists
